### PR TITLE
Removed RHS dependancy; added Insignias; Changes to Kitbag

### PR DESCRIPTION
**Pull request by Synch2k**
Removed RHS eagle backpack and therefore RHS dependency

Added flag insignias for countries where joint operation groups are located (+USA) for usage in mission scenarios and events

Adjusted Kitbag - Medic version now uses Red Crystal as usual in vanilla Arma 3; Description adapted to Vanilla Arma 3

### CODE REVIEW

File: addons/equipment/config.cpp [hunk 1]
```diff
@@ -15,3 +15,4 @@ class CfgPatches {
 };
 
 #include "cfgVehicles.hpp"
+#include "cfgUnitInisgnia.hpp"
```
[the remark this answers is not in the document]
Comment (Synch2k) [reply addons/equipment/config.cpp]: Jup. Danke, ist gefixed. : https://github.com/gruppe-adler/gruppe_adler_mod/pull/90/commits/958fb5c59b9f3ba0d373d63c7e5888c508fe82fa

**Comment by nomisum** [comment]
I dont see any syntax issues anymore. Is this tested already? I wanna update GA mod in repo end of week.

**Comment by Synch2k** [comment]
this exact version was not tested because I am not sure how to test / build it with these x paths ("x\grad\addons\...."), so I requested the check from @McDiod;

**Comment by Synch2k** [comment]
Well, nope. Building seems to work fine, until I try to test it. Then I get the following error, which I think is not related to my changes:
image
So when someone finds themselves knowing what they're doing - please go ahead. @nomisum @McDiod

**Comment by Fusselwurm** [comment]
@Synch2k you need the virtual P: drive with the extracted cba and ace sources. or https://github.com/Dahlgren/DokanPbo

**Comment by Synch2k** [comment]
@Fusselwurm Eh, whatever it is it seems like a unreasonable amount of work for testing one pull request per year 😬 
I tested the content itself before creating the pull request. If I amended the path correctly everything should show up fine. So I guess if someone else could build the mod and check the arsenal for following content that would be great:
Insignias: 
- Germany 🇩🇪
- Sweden 🇸🇪 
- UK 🇬🇧 
- Italy 🇮🇹 
- Spain 🇪🇸 
- USA 🇺🇸 
 Also check if the Kitbag (Black, Medic) has the Red Crystal. Done.

Anyone? Please? 😄

**Comment by McDiod** [comment]
[@gruppe_adler_mod_pr90.zip](https://github.com/gruppe-adler/gruppe_adler_mod/files/2346320/gruppe_adler_mod_pr90.zip)

Testing now. Building only worked with a newer armake version. I'll add it.

File: addons/equipment/cfgUnitInsignia.hpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
+class CfgUnitInsignia
+{
+	class ga_insignia_de
+	{
+		displayName = "Germany"; // Name displayed in Arsenal
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "x\grad\addons\equipment\data\insignia\de_v2.paa"; // Image path
```
Comment (McDiod) [inline addons/equipment/cfgUnitInsignia.hpp]: Yea, I'm getting the `x\grad\addons\...` error when opening the dynamic groups menu to test the insignia. It's probably only missing the leading `\` in the path, like this:

```
texture = "\x\grad\addons\equipment\data\insignia\de_v2.paa";
```

We have some cfgUnitInsignia stuff here:
https://github.com/gruppe-adler/gruppe_adler_mod/blob/master/addons/dynGroups/cfgUnitInsignia.hpp

We can either move all this stuff to `dynGroups`, or move the `dynGroups` stuff to `equipment`. I don't care, it fits both equally well imo.

Backpacks seem to work.

Comment (Synch2k) [reply addons/equipment/cfgUnitInsignia.hpp]: see https://github.com/gruppe-adler/gruppe_adler_mod/pull/90/commits/522060449379fc19de3958bc49dabfa1eb6a3771

**Comment by nomisum** [comment]
@McDiod do you want this
`We can either move all this stuff to dynGroups, or move the dynGroups stuff to equipment. I don't care, it fits both equally well imo.`

resolved in this PR?

**Comment by McDiod** [comment]
Yes. The "noInsignia" insignia is directly tied to the dynGroups addon, so lets just move all of these insignia to dynGroups since we can't do it the other way around.

**Comment by Synch2k** [comment]
I almost forgot. I don't think it's a good idea to merge both addons because they both want to achieve different things. Is there another advantage to combining both?

**Comment by McDiod** [comment]
Keep the backpacks here, move the insignia is what I'm saying. Just seems more sensible to me to *not* add insignia in two different addons. I'm not gonna insist on it though.

File: addons/equipment/cfgUnitInsignia.hpp [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "\x\grad\addons\equipment\data\insignia\es_v2.paa"; // Image path
+		textureVehicle = ""; // Does nothing currently, reserved for future use
+	};
+	
+	class ga_insignia_it
+	{
+		displayName = "Italy"; // Name displayed in Arsenal
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "\x\grad\addons\equipment\data\it_v2.paa"; // Image path
```
Comment (McDiod) [inline addons/equipment/cfgUnitInsignia.hpp]: insignia folder is missing in path

Comment (McDiod) [reply addons/equipment/cfgUnitInsignia.hpp]: This is the actual error in the path. Here and on all insignia below this point.

**Comment by Synch2k** [comment]
Yes, I understood your intention. But dynGroups is a functionality change / improvement of a system, while equipment adds a bunch of random stuff. I don't know. If you put content from one into the other, you get content that you don't necessarily expect. But I also don't really care that was just a thought that popped into my mind when you mentioned it.

File: addons/equipment/cfgUnitInsignia.hpp (hunk 3 above)
Comment (Synch2k) [reply addons/equipment/cfgUnitInsignia.hpp]: 🤦‍♂️ My Copy & Paste skills = -9001 
fixed:
https://github.com/gruppe-adler/gruppe_adler_mod/pull/90/commits/73f2b05e47be7884d19706579b8de19ac4e4a628

**Comment by McDiod** [comment]
Can be merged, not sure what Travis is on about.

**Pull request merged**